README.rst

```diff
@@ -4,3 +4,88 @@ Kinto python client
 Python library to interact with a `Kinto <https://kinto.readthedocs.org>`_
 HTTP server.
 
+Usage
+=====
+
+.. warning::
+    The approach this library takes is different from the one took in the
+    `Cliquetis <https://github.com/mozilla-services/cliquetis>`_ javascript
+    client: Operations are always performed directly on the server, and no
+    synchronisation features are implemented yet.
+
+- Every operation is done remotely; No synchronisation operation is supported;
+- 
+
+To create a bucket and a collection (needed to use them later):
+
+.. code-block:: python
+
+    import kintoclient import Bucket
+
+    # First, create a bucket.
+    bucket = Bucket(name='payments', server_url='http://localhost:8888/v1',
```

Comment:
If we consider that we always interact with one bucket, there is no reason to have a session object (e.g. what we've said yesterday)